src/tests/SearchResults.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  const validProps = {
+    results: ["i1", "i2, i3"],
```

Comment:
Should this props object be `["i1", "i2", "i3"]`, so 3 elements all comma separated?